### PR TITLE
Update shaders.md

### PR DESCRIPTION
**Pull request by Hixie**
Various tweaks to improve the flow based on some areas I was confused about.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]
FYI @craiglabenz

File: src/development/ui/advanced/shaders.md [hunk 1]
```diff
@@ -5,51 +5,65 @@ short-title: Fragment shaders
...
-You can use custom shaders to provide rich graphical effects beyond those
-provided by the Flutter SDK.
+Custom shaders can be used to provide rich graphical effects
+beyond those provided by the Flutter SDK.
+A shader is a program authored in a small, Dart-like language,
+known as GLSL,
+and executed on the user's GPU.
+
+User-authored shaders are added to a Flutter project
+by listing them in the `pubspec.yaml` file,
```
Comment (jonahwilliams) [inline src/development/ui/advanced/shaders.md:20]: Nit, here user refers to the end user, but the next line user refers to the developer.

Maybe specifitcally call out end-user for the first one?

**Review by jonahwilliams — APPROVED** [review]
LGTM, thank you!

File: src/development/ui/advanced/shaders.md (hunk 1 above)
Comment (Hixie) [reply src/development/ui/advanced/shaders.md:20]: i'll replace it with "Custom" for consistency with the rest.

Normally we use "developers" for our users.

**Pull request merged**